crates/voicevox_core_node_api/Cargo.toml

```
@@ -0,0 +1,21 @@
+[package]
+edition = "2021"
+name = "voicevox_core_node_api"
+version = "0.0.0"
+
+[lib]
+crate-type = ["cdylib"]
+
```

<https://github.com/VOICEVOX/voicevox_core/pull/725#issuecomment-2123674059>で思い出しましたが、C/Python/Java APIと同様のこれも入れた方がよいと思います。

```suggestion

[features]
cuda = ["voicevox_core/cuda"]
directml = ["voicevox_core/directml"]
```

---

この提案を採用すると「the package `voicevox_core_node_api` depends on `voicevox_core`, with features: `cuda` but `voicevox_core` does not have these features.」と怒られてしまいます…
他言語APIのパッケージには `cuda = ["voicevox_core/cuda"]` の行はないようなので、
```suggestion

[features]
directml = ["voicevox_core/directml"]
```
こうでしょうか？

---

あ、`cuda`は<https://github.com/VOICEVOX/voicevox_core/pull/725#issuecomment-2123674059>で追加されたものです!

---

なるほど
mainブランチに追従したら通りました！